youtube/generic.md

```diff
@@ -1,49 +1,57 @@
-...
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
 
+MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!
 
-## LEARN MORE
+## YouTube Shorts
+I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1
 
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
+## Newsletter
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls
 
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
+ 
+\1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+\2. Choose The Issue Type Of Invite Me To The Organization
+\3. Choose The Issue Type Of Invite Me To The Organization
+\4. Open The Issue
+\5. You get an invitation to the community by our GitHub Action 🎉
 
-## SERVICES I USE AND RECOMMEND
+---
 
-Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
-
-- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
-- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
-- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
-- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
-
-## COMMUNITY SPONSORS
+## Community Sponsors
 
-A BIG thank you to my GitHub Sponsors:
-***
 Stephen Mount http://github.com/stemount
 Nicholas Carrigan https://github.com/nhcarrigan
-***
+Santosh Yadav https://github.com/santoshyadavdev
```

Comment:
while copy pasting, all these bullets get added to the same line.
A solution might be to split them into new lines -
```suggestion
\- Santosh Yadav https://github.com/santoshyadavdev
```

Reply:
somewhat like this-
```
\- Santosh Yadav https://github.com/santoshyadavdev
\- Stephen Mount http://github.com/stemount
```
\- Santosh Yadav https://github.com/santoshyadavdev
\- Stephen Mount http://github.com/stemount